flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDbStateDataTransfer.java

```diff
@@ -61,6 +69,88 @@ static void transferAllStateDataToDirectory(
 		downloadDataForAllStateHandles(miscFiles, dest, restoringThreadNum, closeableRegistry);
 	}
 
+	public static void uploadFilesToCheckpointFs(
+		@Nonnull Map<StateHandleID, Path> files,
+		int numberOfSnapshottingThreads,
+		CheckpointStreamFactory checkpointStreamFactory,
+		CloseableRegistry closeableRegistry,
+		Map<StateHandleID, StreamStateHandle> hanldes) throws Exception {
```

Comment:
@azagrebin The implementation here returns `handles` implicitly because I thought the caller of the function `uploadFilesToCheckpointFs` may know the size of `handles`, and could init the map by `new HashMap(size)`. 
I agree that the map's size here would not be too large because we'are under incremental mode, I'll change to the explicitly mode. Anyway, I'll add a java doc for these two public function.